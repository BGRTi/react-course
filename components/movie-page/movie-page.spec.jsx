import React from 'react';
import { shallow } from 'enzyme';

import MoviePageContainer from './movie-page';

function setup() {
  const wrapper = shallow(<MoviePageContainer />);
  return { wrapper };
}

describe('MoviePageContainer Test Suite', () => {
  it('render a MoviePageContainer', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });

  it('Should have an image', () => {
    const { wrapper } = setup();
    expect(wrapper.find('.movie-page').exists()).toBe(true);
  });
});
