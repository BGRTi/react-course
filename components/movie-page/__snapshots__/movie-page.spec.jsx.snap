// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageContainer Test Suite render a MoviePageContainer 1`] = `ShallowWrapper {}`;
